spec/support/corpus.rb

```diff
@@ -131,7 +131,7 @@ def install_mutant
         end
         lockfile = repo_path.join('Gemfile.lock')
         lockfile.delete if lockfile.exist?
-        system('bundle install')
+        system(%w[bundle install])
```

Comment:
The `install` comment is optional. You could just say `%w[bundle]` if you wanted.